utils/src/main/scala/za/co/absa/enceladus/utils/general/ProjectMetadata.scala

```diff
@@ -16,8 +16,11 @@
 package za.co.absa.enceladus.utils.general
 
 trait ProjectMetadata {
-
   def projectVersion: String = {
     getClass.getPackage.getImplementationVersion
   }
+
+  def projectMajorVersion: Int = {
```

Comment:
Would a check make sense here?

Reply:
In the next commit, this will be deleted/reverted

Reply:
Fixed